src/main/java/com/adobe/aem/compgenerator/utils/HTMLUtils.java

```diff
@@ -0,0 +1,115 @@
+package com.adobe.aem.compgenerator.utils;
+
+import com.adobe.aem.compgenerator.Constants;
+import com.adobe.aem.compgenerator.models.GenerationConfig;
+import com.adobe.aem.compgenerator.models.Property;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class HTMLUtils {
+
+    public static String renderHtml(GenerationConfig generationConfig) {
+        List<Property> props = getAllProperties(generationConfig);
+        String slingModel = StringUtils.uncapitalize(generationConfig.getJavaFormatedName()) + "Model";
+        StringBuilder renderedHtml = new StringBuilder();
+
+        for (Property prop : props) {
+            String label = getLabel(prop);
+
+            if (Constants.TYPE_CHECKBOX.equals(prop.getType())) {
+                label = prop.getAttributes().get("text");
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        " ? 'checked' : 'unchecked'"
+                ));
+            } else if (Constants.TYPE_DATEPICKER.equals(prop.getType())) {
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        ".time.toGMTString"
+                ));
+            } else if (Constants.TYPE_IMAGE.equals(prop.getType())) {
+                renderedHtml.append(generateImageHtml(label, prop.getField(), slingModel));
+            } else if (Constants.TYPE_MULTIFIELD.equals(prop.getType())) {
+                renderedHtml.append(generateListHtml(prop, slingModel));
+            } else {
+                renderedHtml.append(generateParagraphHtml(label,
+                        prop.getField(),
+                        slingModel,
+                        StringUtils.EMPTY));
+            }
+        }
+        return renderedHtml.toString();
+    }
+
+    private static String getLabel(Property prop) {
```

Comment:
Your demo code reveals that we forgot to put a label for the radio button example in the demo config file.

Can you please add `"label": "Radio Test",` to the radio button example in `data-config.json`?

Reply:
Added